public/data/ckvignesh.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "C K Vignesh",
+  "bio": "Programmer",
+  "avatar": "https://github.com/ckvignesh.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ckvignesh",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vigneshCodes",
+      "icon": "twitter"
+    },
+    {
+        "name": "My Open Source Contributions",
+        "url": "https://ckvignesh.notion.site/List-of-Open-Source-Contributions-09b2368ec25742a0b9f0421a0d1c9f0c",
+        "icon": "github"
```

Comment:
Github logo here?

Reply:
Sure, I'll change it. Do you have any suggestions for it from the existing icons? Thanks,

Reply:
I'll make it a link icon.

Reply:
Thanks, @schmelto